huawei/integrationpluginhuawei.json

```diff
@@ -0,0 +1,297 @@
+{
+    "name": "Huawei",
+        "displayName": "Huawei FusionSolar",
+        "id": "fc3e4509-47f3-4622-9bc4-0a90fe2b6262",
+        "vendors": [
+        {
+            "name": "huawei",
+            "displayName": "Huawei",
+            "id": "f654c99d-a286-4abb-b33e-1a71843d8da0",
+            "thingClasses": [
+                {
+                    "name": "huaweiInverter",
+                    "displayName": "Huawei FusionSolar Inverter",
```

Comment:
AFAIU, there will be two options for this device, Modbus TCP (this one) and a Modbus RTU given that this Modbus TCP adapter is not really good. Is that correct? If so, perhaps this should already mention something like `Huawei FusionSolar (Network based)` so the other would be `Huawei FusionSolar (Serial based)` or something like this?